packages/zwave-js/src/lib/commandclass/SceneActuatorConfigurationCC.ts

```diff
@@ -0,0 +1,395 @@
+				throw new ZWaveError(
+					`scene id ${options.sceneId} out of range [1..255]`,
+					ZWaveErrorCodes.Argument_Invalid,
+				);
+			}
+			this.sceneId = options.sceneId;
+			this.dimmingDuration = options.dimmingDuration;
+			if (options.level != undefined) {
+				if (options.level < 0 || options.level > 255) {
+					throw new ZWaveError(
+						`level ${options.level} out of range [1..255]`,
+						ZWaveErrorCodes.Argument_Invalid,
+					);
+				}
+				this.override = true;
+				this.level = options.level;
+			} else {
+				this.override = false;
+				this.level = 0xff; // will be ignored
+			}
+		}
+	}
+
+	public sceneId: number;
+	public dimmingDuration: Duration;
+	public override: boolean;
+	public level: number;
+
+	public serialize(): Buffer {
+		this.payload = Buffer.from([
+			this.sceneId,
+			this.dimmingDuration.serializeSet(),
+			this.override ? 0b1000_0000 : 0x00,
+			this.level,
+		]);
+		return super.serialize();
+	}
+}
+
+@CCCommand(SceneActuatorConfigurationCommand.Report)
+export class SceneActuatorConfigurationCCReport extends SceneActuatorConfigurationCC {
+	public constructor(
+		driver: Driver,
+		options: CommandClassDeserializationOptions,
+	) {
+		super(driver, options);
+		validatePayload(this.payload.length >= 3);
+		this.sceneId = this.payload[0];
+		this.level = this.payload[1];
+		this.dimmingDuration =
+			Duration.parseReport(this.payload[2]) ?? new Duration(0, "unknown");
+
+		this.persistValues();
+	}
+
+	public sceneId: number;
+	public level: number;
+	public dimmingDuration: Duration;
+
+	public persistValues(): boolean {
+		persistSceneActuatorConfig.call(
+			this,
+			this.sceneId,
+			this.level,
+			this.dimmingDuration,
+		);
+		return true;
+	}
+
+	public toLogEntry(): MessageOrCCLogEntry {
+		return {
+			...super.toLogEntry(),
+			message: {
+				sceneId: this.sceneId,
+				level: this.level,
+				dimmingDuration: this.dimmingDuration.toString(),
+			},
+		};
+	}
+}
+
+function testResponseForSceneActuatorConfigurationGet(
+	sent: SceneActuatorConfigurationCCGet,
+	received: SceneActuatorConfigurationCCReport,
+) {
+	return received.sceneId === sent.sceneId;
+}
+
+interface SceneActuatorConfigurationCCGetOptions extends CCCommandOptions {
+	sceneId: number;
+}
+
+@CCCommand(SceneActuatorConfigurationCommand.Get)
+@expectedCCResponse(
+	SceneActuatorConfigurationCCReport,
+	testResponseForSceneActuatorConfigurationGet,
+)
+export class SceneActuatorConfigurationCCGet extends SceneActuatorConfigurationCC {
+	public constructor(
+		driver: Driver,
+		options:
+			| CommandClassDeserializationOptions
+			| SceneActuatorConfigurationCCGetOptions,
+	) {
+		super(driver, options);
+		if (gotDeserializationOptions(options)) {
+			// TODO: Deserialize payload
+			throw new ZWaveError(
+				`${this.constructor.name}: deserialization not implemented`,
+				ZWaveErrorCodes.Deserialization_NotImplemented,
+			);
+		} else {
+			if (options.sceneId < 1 || options.sceneId > 255) {
```

Comment:
> The value 0 MUST indicate that the current active scene (if any).

So 0 is also okay. I think this is actually just the same as in `Scene Controller Configuration CC` including the predicate function.

Reply:
Yeah, I rushed this one and didn't read the specs carefully.